### ISSUE TITLE
Gitlab README Card Markdown Display Problem

### ISSUE THREAD

**Issue by eceeeren** [immobiliare/backstage-plugin-gitlab#327]
Hello everyone,

We have this plugin added in our Backstage dashboard and using EntityGitlabReadmeCard component as well. However the markdown display is not working well, as you could see in the picture. Any ideas to fix it?

Best

![markdown](https://github.com/immobiliare/backstage-plugin-gitlab/assets/23586149/9287bd0c-7c01-4107-a8cc-43dab48cafe1)

**Comment by antoniomuso** [comment]
Hi @eceeeren, can you post the Markdown?

**Comment by eceeeren** [comment]
Hello, here it is:

[README.md](https://github.com/immobiliare/backstage-plugin-gitlab/files/13265353/README.md)

**Comment by eceeeren** [comment]
Hello everyone,

I found the css lines that were causing the issue:

`h1, h2, h3, h4, h5, h6 {
     font-size: 100%;
    font-weight: var(--pf-global--FontWeight--normal);
}

html, body, p, ol, ul, li, dl, dt, dd, blockquote, figure, fieldset, legend, textarea, pre, iframe, hr, h1, h2, h3, h4, h5, h6 {
    padding: 0;
    margin: 0;
}`

It is in node_modules\@patternfly\react-core\dist\styles\base.css. But I am not sure how to approach this problem, Should I override these?

**Comment by antoniomuso** [comment]
This plugin uses the core-component Backstage plugin. The bug is probably related to it.

**Comment by eceeeren** [comment]
Any ideas on how to fix it?

**Comment by antoniomuso** [comment]
I am trying to understand why it works locally for me. I made a new prerelease, can you try it? [v6.4.0-alpha.1](https://github.com/immobiliare/backstage-plugin-gitlab/releases/tag/v6.4.0-alpha.1)

**Comment by eceeeren** [comment]
Hello, I am sorry for the confusion, I think this happens because of a special package that we have and it is not a general issue.

Now I am trying to override BackstageMarkdownContent but it says: "The expected type comes from property 'components' which is declared here on type 'UnifiedThemeOptions'". I thought it was fixed in this issue:

https://github.com/backstage/backstage/issues/18422

Any idea on how to apply this? I did "yarn upgrade" but it didn't work. Or should I wait until it is released 🤔
